### PR TITLE
Add ping6 command

### PR DESCRIPTION
**Pull request by brknkfr**
- [x] I read [CONTRIBUTING.md](https://github.com/Cyclenerd/static_status/blob/master/CONTRIBUTING.md)
- [x] I used tabs to indent
- [x] I checked my code with [ShellCheck](https://www.shellcheck.net/)

## Notes

Sometimes I want to check a hostname for IPv4 _and_ IPv6 connection.

### CODE REVIEW

File: status.sh [hunk 1]
```diff
@@ -750,6 +758,32 @@ while IFS=';' read -r MY_COMMAND MY_HOSTNAME_STRING MY_PORT || [[ -n "$MY_COMMAN
...
+		(( MY_HOSTNAME_COUNT++ ))
+		# Detect ping6 Version
+		ping6 &> /dev/null
+		# FreeBSD: 64 = ping6 -n -t TIMEOUT
+		# macOS:   64 = ping6 -n -t TIMEOUT
+		# GNU:      2 = ping6 -n -w TIMEOUT (-t TTL)
+		# OpenBSD:  1 = ping6 -n -w TIMEOUT (-t TTL)
+		if [ $? -gt 2 ]; then
+			# BSD ping6
+			MY_PING6_COMMAND='ping -n -t'
```
Comment (Cyclenerd) [inline status.sh:771]: s/ping/ping6/

**Comment by Cyclenerd** [comment]
The command still uses ping without 6. Can you also resolve the conflict. Thank you very much 👍

File: status.sh (hunk 1 above)
Comment (brknkfr) [reply status.sh:771]: Oh!

**Comment by Cyclenerd** [comment]
Thank you once again.

**Pull request merged**